.github/workflows/ci-cd.yml

```
@@ -150,6 +150,7 @@ jobs:
     strategy:
       matrix:
         pyver:
+        - "3.13-freethreading"
```

let's follow the existing convention of not adding unnecessary quoting
```suggestion
        - 3.13-freethreading
```